client/landing/stepper/declarative-flow/sensei.scss

```diff
@@ -0,0 +1,6 @@
+$sensei-mobile-layout-width: 1200px;
```

Comment:
As a small issue of code organization (possibly personal preference on this), part of me would like to find a better place to put this scss file. It's kind of by itself here alongside all the flow defintiations. Most obvious place would be stepper/declarative-flow/internals/global.scss. There is already some flow specific scss in that file that's intended to apply to all steps in a particular flow.

Reply:
> As a small issue of code organization (possibly personal preference on this), part of me would like to find a better place to put this scss file

Indeed this was initially inside the `internals` directory but we moved it outside it as part of another review :) As the file is sourced by both JS and CSS files it would be more complex to move the CSS inside `global.scss`. For this reason, I just moved it inside the internals directory, which is what other flows are doing too.